src/main/java/org/sonarlint/intellij/util/HelpLabelUtils.kt

```diff
@@ -26,12 +26,26 @@ import org.sonarlint.intellij.telemetry.LinkTelemetry
 
 class HelpLabelUtils {
     companion object {
+        private var currentFileText: String = "SonarLint analyzes in real-time the active file only and reports any issues found in this view. ";
```

Comment:
Do you know if this sentence was reviewed? I'd personally go with:
> SonarLint only analyzes the active file in real-time and reports any issues found in this view.
WDYT?

Reply:
Yes, we agreed on this sentence with Marco

Reply:
I think it's not super well written, at least add a comma before the "and" maybe 😄 

> "SonarLint analyzes in real-time the active file only, and reports any issues found in this view. "